tools/Raven.Debug/Dump/Dumper.Linux.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Microsoft.Diagnostics.Tools.Dump
+{
+    public partial class Dumper
+    {
+        private static class Linux
+        {
+            internal static async Task CollectDumpAsync(Process process, string fileName, DumpTypeOption type)
+            {
+                // We don't work on WSL :(
+                string ostype = await File.ReadAllTextAsync("/proc/sys/kernel/osrelease");
```

Comment:
why is that?

Reply:
It's copied from https://github.com/dotnet/diagnostics